platform/linux/build.gradle

```diff
@@ -61,6 +62,22 @@ buildDeb {
     preUninstall file('src/deb/preUninstall.sh')
 }
 
+docker {
+    name "trellisldp/trellis:${project.version}"
+    tags 'latest'
+    dockerfile file('src/docker/Dockerfile')
+    files tasks.distTar.outputs, './src/dist/etc/config.yml','./src/docker/command.sh'
```

Comment:
I think this line must be revised. It refers to the outputs of the "distTar" task, which I am pretty sure does not exist in stock trellis, at least I don't see it here.. The error I get upon Docker startup is that it cannot find "trellis" on the path.. "trellis" would have been packaged at /opt/trellis/bin/trellis if things had worked correctly.

Reply:
Ok, the `distTar` task _should_ be present here, but I'll need to look at this more carefully.